Lib/symtable.py

```diff
@@ -66,23 +77,23 @@ def __repr__(self):
     def get_type(self):
         """Return the type of the symbol table.
 
-        The values returned are 'class', 'module', 'function',
-        'annotation', 'TypeVar bound', 'type alias', and 'type parameter'.
+        The values returned are one of the values in
+        the ``SymbolTableType`` enumeration.
         """
         if self._table.type == _symtable.TYPE_MODULE:
-            return "module"
+            return SymbolTableType.MODULE.value
```

Comment:
Why not this?
```suggestion
            return SymbolTableType.MODULE
```
It still returns a string equal to `"module"`, so this should be fine.

Reply:
Good catch! my bad.